clr.coreclr.props

```diff
@@ -101,4 +101,8 @@
 
     <FeatureCoreFxGlobalization>true</FeatureCoreFxGlobalization>
   </PropertyGroup>
+
+  <PropertyGroup>
```

Comment:
I meant to exclude this for Unix- thanks for catching it. I'll update it with the condition.